sphinx/texinputs/latexmkrc_t

```diff
@@ -1,5 +1,13 @@
+{% if latex_engine == 'pdflatex' -%}
 $latex = 'latex ' . $ENV{'LATEXOPTS'} . ' %O %S';
 $pdflatex = 'pdflatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
+{% elif latex_engine == 'lualatex' -%}
+$latex = 'lualatex --output-format=dvi ' . $ENV{'LATEXOPTS'} . ' %O %S';
+$pdflatex = 'lualatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
+{% elif latex_engine == 'xelatex' -%}
+$latex = 'xelatex --no-pdf ' . $ENV{'LATEXOPTS'} . ' %O %S';
+$pdflatex = 'xelatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
+{% endif -%}
 $lualatex = 'lualatex ' . $ENV{'LATEXOPTS'} . ' %O %S';
 $xelatex = 'xelatex --no-pdf ' . $ENV{'LATEXOPTS'} . ' %O %S';
```

Comment:
these Perl variables are defined for the case latexmk is a recent version, then user can employ ``LATEXMKOPTS="-xelatex"`` for example, so this will be used, and have the benefit of faster builds (PDF only on last run).

with earlier latexmk versions, they will be ignored anyhow.